tests/test.py

```diff
@@ -1394,17 +1302,11 @@ def _fork_acra(self, acra_kwargs, popen_kwargs):
         if self.LOG_METRICS:
             args['incoming_connection_prometheus_metrics_string'] = self.get_prometheus_address(
                 self.ACRASERVER_PROMETHEUS_PORT)
-        if self.CONNECTOR_TLS_TRANSPORT:
-            args['acraconnector_tls_transport_enable'] = 'true'
         if self.with_tls():
             args['tls_key'] = TEST_TLS_SERVER_KEY
             args['tls_cert'] = TEST_TLS_SERVER_CERT
             args['tls_ca'] = TEST_TLS_CA
             args['tls_auth'] = ACRA_TLS_AUTH
-            args['tls_ocsp_url'] = 'http://localhost:{}'.format(self.OCSP_SERVER_PORT)
```

Comment:
why we don't use it now? if it worked before with `self.with_tls`, what problem do we have now?

Reply:
Returned it back, It works, i think kind of typo, thanks